h2o-core/src/main/java/hex/Model.java

```diff
@@ -5,6 +5,8 @@
 import hex.genmodel.easy.RowData;
 import hex.genmodel.easy.exception.PredictException;
 import hex.genmodel.easy.prediction.*;
+import org.apache.commons.compress.archivers.zip.ZipArchiveEntry;
```

Comment:
Can you quickly explain why are you using ZipArchiveEntry instead of java `java.util.zip.ZipEntry` ? What are benefits?